### PR TITLE
Sound Library Updates

### PR DESCRIPTION
**Pull request by epeach**
Small styles changes to Sound Library Dialog.

*Displays (from L to R) 'Sound Library' tab before 'My Files'
*Updates 'My Files' to 'Make new sounds'
*Increments study value in log to differentiate logs after this change

screenshot from 2018-09-12 15-36-42

### CODE REVIEW

File: apps/src/code-studio/components/SoundPicker.jsx [hunk 1]
```diff
@@ -94,7 +95,7 @@ export default class SoundPicker extends React.Component {
 
     firehoseClient.putRecord(
       {
-        study: 'sound-dialog',
+        study: 'sound-dialog-1M',
```
Comment (epeach) [inline apps/src/code-studio/components/SoundPicker.jsx]: Extra 'M' slipped in - update coming.

File: apps/i18n/common/en_us.json [hunk 2]
```diff
@@ -770,6 +770,7 @@
   "loginTypeWordResetDescription": "You may reset a student's secret words at any time by choosing Show secret and then Reset secret. A new pair of secret words will be generated for that student to use when they sign in.",
   "loginTypeWordLongDescription": "This section uses word as its secret type. It means that each of your students has a secret pair of words used in place of a password to sign in. Students should use the sign in web address given above to sign in.",
   "makeMyOwn": "Make my own",
+  "makeNewSounds": "Make new sounds",
```
Comment (Erin007) [inline apps/i18n/common/en_us.json:773]: teeny tiny nit: can we move this string so it's fully alphabetical?

File: apps/src/code-studio/components/SoundListEntry.jsx [hunk 3]
```diff
@@ -69,7 +69,7 @@ class SoundListEntry extends React.Component {
       this.setState({isPlaying: true});
       firehoseClient.putRecord(
         {
-          study: 'sound-dialog',
```
Comment (Erin007) [inline apps/src/code-studio/components/SoundListEntry.jsx:72]: Do we expect this study name to change again? If so, might be worth putting in a variable so we only need to change in one place, but fine as is too.

Comment (epeach) [reply apps/src/code-studio/components/SoundListEntry.jsx:72]: As far as I'm aware - we do not expect it to change again.

File: apps/i18n/common/en_us.json (hunk 2 above)
Comment (epeach) [reply apps/i18n/common/en_us.json:773]: I think this one is alpha-ordered? But I'll move 'SoundLibrary' up one to be alpha-ed.

**Pull request merged**